### PR TITLE
Created basic map view

### PR DESCRIPTION
**Pull request by jk1257**
Added the basic map view so we can start plotting points

### CODE REVIEW

File: templates/maps/default.html [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  <head>
+    <title>Interactive maps for Django web apps</title>
+    <script src='https://api.mapbox.com/mapbox-gl-js/v2.3.1/mapbox-gl.js'></script>
+<link href='https://api.mapbox.com/mapbox-gl-js/v2.3.1/mapbox-gl.css' rel='stylesheet' />
+  </head>
+  <body>
+   <h1>Map</h1>
+   <div id='map' style='width: 100%; height: 400px;'></div>
+   <script>
+     mapboxgl.accessToken = 'pk.eyJ1IjoiamsxMjU3IiwiYSI6ImNrdXJrZzJlcDNvanIzMHBodXB3eWphYWwifQ.JcoTmAWXZ__t64PfaR7PUg';
```
Comment (susmitabhowmik) [inline templates/maps/default.html:12]: I think we want to store this access token as a .env variable

File: maps/urls.py [hunk 2]
```diff
@@ -0,0 +1,6 @@
+from django.conf.urls import url                                                                                                                              
+from . import views
+
+urlpatterns = [ 
+    url(r'^maps/', views.default_map, name="maps"),
+]
```
Comment (susmitabhowmik) [inline maps/urls.py:6]: Minor change but looks like git wants a new line at the end of this file, which is why the red circle with the minus sign is there

File: maps/views.py [hunk 3]
```diff
@@ -0,0 +1,10 @@
+from django.shortcuts import render
+
+# Create your views here.
+def default_map(request):
+    # TODO: move this token to Django settings from an environment variable
+    # found in the Mapbox account settings and getting started instructions
+    # see https://www.mapbox.com/account/ under the "Access tokens" section
+    mapbox_access_token = 'pk.my_mapbox_access_token'
+    return render(request, 'default.html', 
+                  { 'mapbox_access_token': mapbox_access_token })
```
Comment (susmitabhowmik) [inline maps/views.py:10]: Same as above, think it wants a new line here too

**Review by susmitabhowmik — COMMENTED** [review]
This looks great!! 🎉  Awesome work with connecting to Mapbox and creating the maps page. 

I left some comments for some minor changes. A few other minor changes I think we should consider are adding the `.DS_Store` and `config/.DS_Store` files to the `.gitignore`. These are files automatically generated by apple and we don't need to add them to our application.

I am also wondering if we should add a section to our ReadMe with routes? There is no real way currently to know which routes are available within the application other than looking at the `url.py` file. Let me know what you think!

**Comment by jk1257** [comment]
> This looks great!! 🎉 Awesome work with connecting to Mapbox and creating the maps page.
> 
> I left some comments for some minor changes. A few other minor changes I think we should consider are adding the `.DS_Store` and `config/.DS_Store` files to the `.gitignore`. These are files automatically generated by apple and we don't need to add them to our application.
> 
> I am also wondering if we should add a section to our ReadMe with routes? There is no real way currently to know which routes are available within the application other than looking at the `url.py` file. Let me know what you think!

**Pull request merged**